.github/workflows/validate-licenses.yaml

```diff
@@ -32,9 +30,42 @@ jobs:
           ./bloodhound --no-validation --list-images > images.txt
           echo "printing contents of images.txt"
           cat images.txt
+      - name: Update licenses
+        id: updateLicenses
+        continue-on-error: true
```

Comment:
i think we might want to remove `continue-on-error: true`, if this step errors i don't think there will be anything to commit